extension/scripts/background.js

```diff
@@ -33,8 +33,8 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
 				)
 			})
 		})
-	} else if (message.row) {
-		const body = JSON.stringify({row: message.row})
+	} else if (message.content) {
+		const body = JSON.stringify({content: message.content})
 		console.log('4. Message body sent: ', body)
 		fetch('http://localhost:3001/send-image', {
```

Comment:
It would be nice to refactor this to async/await at some point.

Reply:
From what I've researched, I would need to use axios instead of fetch? 
Because this is a google extension I would need to make it more complicated adding 'webpack' so I can add 'axios':
["I would recommend using the fetch API as an alternative. It's already freely available in chrome extensions and works in very similar way to axios." ](https://stackoverflow.com/questions/65314853/how-to-add-a-package-such-as-axios-to-a-chrome-extension)

Reply:
Async/await is just syntactic sugar over `.then()`, so it should work. But we can try it later.